Makefile

```diff
@@ -146,6 +148,41 @@ install_tfhe_lints:
 	(cd utils/cargo-tfhe-lints-inner && cargo install --path .) && \
 	cd utils/cargo-tfhe-lints && cargo install --path .
 
+.PHONY: create_venv # Create Python virtualenv under venv/
+create_venv:
+	virtualenv venv
+
+.PHONY: setup_venv # Setup Python virtualenv for wasm tests
+setup_venv:
+	@source venv/bin/activate || \
+	(echo "Virtualenv not available, please create one named 'venv' at the root of this repository." && exit 1 ) && \
+	pip3 install -r ci/webdriver_requirements.txt
+
+install_chrome_browser: url = "https://storage.googleapis.com/chrome-for-testing-public/128.0.6613.137/linux64/chrome-linux64.zip"
+install_chrome_browser: checksum = "c5d7da679f3a353ae4e4420ab113de06d4bd459152f5b17558390c02d9520566"
+install_chrome_browser: dest = "$(WEB_RUNNER_DIR)/chrome"
+install_chrome_browser: filename = "chrome-linux64.zip"
+
+install_chrome_web_driver: url = "https://storage.googleapis.com/chrome-for-testing-public/128.0.6613.137/linux64/chromedriver-linux64.zip"
+install_chrome_web_driver: checksum = "f041092f403fb7455a6da2871070b6587c32814a3e3c2b0a794d3d4aa4739151"
+install_chrome_web_driver: dest = "$(WEB_RUNNER_DIR)/chrome"
+install_chrome_web_driver: filename = "chromedriver-linux64.zip"
```

Comment:
same question about having the variables defined "far" away from the target using them ?